Release/src/pplx/pplx.cpp

```diff
@@ -63,58 +64,34 @@ static struct _pplx_g_sched_t
 {
     typedef std::shared_ptr<pplx::scheduler_interface> sched_ptr;
 
-    _pplx_g_sched_t() { m_state = post_ctor; }
-
-    ~_pplx_g_sched_t() { m_state = post_dtor; }
-
     sched_ptr get_scheduler()
-    {
-        switch (m_state)
+    {        
+        sched_ptr sptr = std::atomic_load_explicit(&m_scheduler, std::memory_order_consume);
```

Comment:
Not sure how I feel about memory_order_consume; even SG1 thinks that thing is effectively broken. Any objections to changing this to acquire?

Reply:
`consume` and `acquire` should be equivalent here as atomic is not used for synchronization here. Anyway, I'll update with `acquire`.